grails-app/domain/streama/Video.groovy

```diff
@@ -12,7 +12,9 @@ class Video {
   Double vote_average
   Integer vote_count
   Double popularity
-
+  int intro_start
+  int intro_end
+  int outro_start
```

Comment:
instead of using int, use Integer. int does not allow for NULL values and therefore they all get initialized with 0 which could potentially break the application.